pyproject.toml

```diff
@@ -75,7 +75,7 @@ dependencies = [
     "Shapely>=2",
     "splink>=3.9.14,<4", # Need to update PUDL to use new Splink v4 API. See issue #3735
     "sphinx>=7.4.4",
-    "sphinx-autoapi>=3",
+    "sphinx-autoapi>=3,<3.2",
```

Comment:
Required because of what seems like a bug in newer `sphinx-autoapi` which I reported here: https://github.com/readthedocs/sphinx-autoapi/issues/476